subsys/settings/src/settings_store.c

```diff
@@ -60,13 +60,34 @@ int settings_load_subtree(const char *subtree)
 	 */
 	k_mutex_lock(&settings_lock, K_FOREVER);
 	SYS_SLIST_FOR_EACH_CONTAINER(&settings_load_srcs, cs, cs_next) {
-		cs->cs_itf->csi_load(cs, subtree);
+		cs->cs_itf->csi_load(cs, subtree, NULL, NULL);
 	}
 	rc = settings_commit_subtree(subtree);
 	k_mutex_unlock(&settings_lock);
 	return rc;
 }
 
+int settings_load_subtree_direct(
```

Comment:
@rako what do you think about making `settings_load_subtree()` the wraper around `settings_load_subtree_direct()`

Reply:
ok, never mind